taglib/dsdiff/dsdifffile.cpp

```diff
@@ -66,7 +66,7 @@ namespace
       return false;
 
     for(int i = 0; i < 4; i++) {
-      if(name[i] < 32 || name[i] > 127)
+      if(name[i] < 32 /*|| name[i] > 127*/)
```

Comment:
Should this not be 126?

Reply:
I had to reference EA IFF 85 to make sure but you're correct.